Automattic/wp-calypso
# Enable vertical flags in build flow in production, stage, and wpcalypso

#### Proposed Changes

This PR enables the flags `signup/site-vertical-step` and `signup/design-picker-generated-designs` in the production, stage, and wpcalypso environments. Keep in mind that these flags were already enabled in development, and horizon environments.

#### Testing Instructions



1. Follow setup instructions in https://github.com/Automattic/wp-calypso/tree/trunk/test/e2e.
2. Set export CALYPSO_BASE_URL=http://calypso.localhost:3000 (see also [this doc](https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/docs/tests_local.md)).
3. Under test/e2e folder, run `yarn jest specs/onboarding/signup__free`.
